pages/integrations/output_framework.rst

```diff
@@ -17,7 +17,7 @@ outbound messages.
 Enterprise Framework Outputs
 ----------------------------
 
-The Enterprise Output Framework provides a number of new Outputs for the various network 
+The Enterprise Output Framework provides a number of new Outputs for various network 
 transport types. All of these Outputs first write messages to an on-disk journal in the 
 Graylog cluster.  Messages stay in the on-disk journal until the Output is able to 
 successfully send the data to the external receiver.
```

Comment:
@waab76 - when is the on-disk journal cleared ?

Reply:
Currently, the on-disk journal is never cleared.  We should add support for clearing the journal when an output is deleted.